src/styles.css

```diff
@@ -0,0 +1,3 @@
+.hidden {
+  display: none;
```

Comment:
@jrmedina @sway3406

Reply:
Looks good, maybe we could add the red border property to all containers for styling too?